src/test/java/org/broadinstitute/hellbender/utils/spark/SparkUtilsUnitTest.java

```diff
@@ -139,17 +141,31 @@ public void testPutReadsWithSameNameInSamePartition(int numPairs, int numPartiti
         return reads;
     }
 
-    @Test(expectedExceptions = GATKException.class)
```

Comment:
Great to see this test changed to passing.

Reply:
👍 Did you run into this in the wild?

Reply:
@lbergelson Yes, I did see the empty partition issue in the wild.  Situation was a very small BAM containing large contigs from a local assembly with lots of supplemental alignments.